core/node/consistency_checker/src/lib.rs

```diff
@@ -380,52 +376,36 @@ impl ConsistencyChecker {
     const DEFAULT_SLEEP_INTERVAL: Duration = Duration::from_secs(5);
 
     pub async fn new(
-        l1_client: Box<DynClient<L1>>,
-        gateway_client: Option<Box<DynClient<L1>>>,
+        gateway_client: Box<dyn EthInterface>,
```

Comment:
is it only gateway client or settlement layer client?

Reply:
fixed